quirks/password-rules.json

```diff
@@ -8,9 +8,15 @@
     "aliexpress.com": {
         "password-rules": "minlength: 6; maxlength: 20;"
     },
+    "allianz.com.br": {
+        "password-rules": "minlength: 4; maxlength: 4;"
```

Comment:
And all ascii-printable characters are allowed? Or are there any restrictions?

Reply:
all ascii-printable allowed.